conf/modules.config

```diff
@@ -197,6 +197,20 @@ process {
             '-outfmt',
             '"6 qseqid ppos"',
         ].join(' ').trim()
+
+        publishDir = [
+            [
+                path: "${params.outdir}",
+                mode: params.publish_dir_mode,
+                failOnError: true,
+                saveAs: {
+                    filename -> {
+                        def output_file = new File(filename);
+                        return "${study_reads_folder()}/assembly/${meta.assembler}/${meta.assembler_version}/decontamination/${output_file.name}";
```

Comment:
would it publish all runs of BLASTN? I guess we need only final one...

Reply:
Yes, I think publishing both contaminated hits are useful. So, this way we can have the contigs that are contaminated with human-phix and the ones with host (reference genome)

Reply:
I like the idea of having both files, but I think that the majority of the contamination happens at the read level. The post-assembly decontamination is usually much less impactful (if at all). I think that having this for both reads_qc and assembly_qc would be nice

Reply:
Alright, I agree. But we don't keep the decontaminated reads.. not sure if we should as it will be storage hungry. What we can do is store some stats about how many reads we discarded. But that this is for another PR